### PR TITLE
add install guide

### PR DESCRIPTION
**Pull request by ChiuWang**

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -14,7 +18,10 @@ $ kubectl label nodes nodeName coredump=true
 run the following command:
 ```shell
 $ cd gencerts
-$ ./make-ca-cert.sh <ip of the host where the coredump-detector is deployed, and kube-apiserver will use this ip to access coredump-detector>
+$ ./make-ca-cert.sh webhook.default.svc <replace this with 1.[alone]- ip of the host where the coredump-detector is deployed, and kube-apiserver will use this ip to access coredump-detector 2.[as service]- the wenhook service DNS name>
+```
+If you choose setup [alone], you should modify `make-ca-cert.sh`, set `use_cn=false`.
```
Comment (CaoShuFeng) [inline INSTALL.md:23]: Why?

File: INSTALL.md [hunk 2]
```diff
@@ -52,19 +59,75 @@ users:
...
 ```
 # the ca.crt, server.cert, server.key are files generated before
 # you should only run this in the host with ip you specified when preparing certificates
 ./coredump-detector -v=10 --alsologtostderr --client-ca-file gencerts/output/ca.crt --tls-cert-file gencerts/output/server.cert --tls-private-key-file gencerts/output/server.key
 ```
 
-# make MutatingWebhookConfiguration object in kube-apiserver
+## Deploy it as a k8s service
+1. Create a secret containing ca key and crt
+`$ kubectl create secret generic pod-secret --from-file=servercert=/path/to/server.cert --from-file=serverkey=/path/to/server.key --from-file=ca=/path/to/ca.crt`
```
Comment (CaoShuFeng) [inline INSTALL.md:79]: --from-file=ca.crt=ca.crt --from-file=server.key=server.key --from-file=server.cert=server.cert

File: INSTALL.md [hunk 3]
```diff
@@ -52,19 +59,75 @@ users:
...
+  selector:
+    matchLabels:
+      app: coredump-detector
+  template:
+    metadata:
+      labels:
+        app: coredump-detector
+    spec:
+      nodeSelector:
+        coredump: "true"
```
Comment (CaoShuFeng) [inline INSTALL.md:113]: Why this must be scheduled to a node that supports coredump?

File: INSTALL.md (hunk 1 above)
Comment (ChiuWang) [reply INSTALL.md:23]: set to `true` means allow to use dns name.

File: INSTALL.md (hunk 2 above)
Comment (ChiuWang) [reply INSTALL.md:79]: reasonable indeed.

**Pull request closed without merging**